test/integration/ttlcontroller/ttlcontroller_test.go

```diff
@@ -60,7 +60,8 @@ func createNodes(t *testing.T, client *clientset.Clientset, startIndex, endIndex
 				},
 			}
 			if _, err := client.CoreV1().Nodes().Create(node); err != nil {
-				t.Fatalf("Failed to create node: %v", err)
+				t.Errorf("Failed to create node: %v", err)
```

Comment:
I'm not sure actually what the correct guidance here should be. I would have expected `t.Fatalf` to be the preferred method over `Errorf` + return.

@kubernetes/sig-testing-pr-reviews

Reply:
Fatalf seems preferable?

Reply:
btw,does it menas I needed not to do some changes,but the staticcheck remain errors.

Reply:
what was the error?

Reply:
I haven't seen the error but fwiw https://github.com/kubernetes/kubernetes/pull/87411 has a similar change; it is clear from that PR that static check seems to want `Errorf` when it occurs in a go routine (which both changes in this PR are).

I thought the same thing initially; that a fatalf seems more intuitive but I guess being in a goroutine it may make other things less predictable?